sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/AzureMonitorTransmitter.cs

```diff
@@ -23,7 +24,7 @@ namespace Azure.Monitor.OpenTelemetry.Exporter
     internal class AzureMonitorTransmitter : ITransmitter
     {
         private readonly ApplicationInsightsRestClient _applicationInsightsRestClient;
-        internal IPersistentStorage _storage;
+        internal PersistentBlobProvider _storage;
```

Comment:
nit: Should we consider renaming the variable? 
It makes difficult to read the code as class is provider and variable is storage.